### PR TITLE
add mixtral-8x7B sagemaker notebooks for trt-llm and lmi-dist

### PR DESCRIPTION
**Pull request by rohithkrn**

### CODE REVIEW

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_mixtral_8x7b.ipynb [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f27a8df",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "instance_type = \"ml.g5.48xlarge\"\n",
+    "# instance_type = \"ml.p4d.24xlarge\"\n",
```
[the remark this answers is not in the document]
Comment (rohithkrn) [reply aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_mixtral_8x7b.ipynb:188]: no, I was testing and forgot to remove

**Review by ydm-amazon — APPROVED** [review]

**Comment by rohithkrn** [comment]
Will merge after the release.

**Comment by zachgk** [comment]
Don't forget to add these notebooks to the index http://docs.djl.ai/docs/demos/aws/sagemaker/large-model-inference/index.html

**Comment by lanking520** [comment]
@rohithkrn just for ease, could you also add a vllm sample?

**Comment by ydm-amazon** [comment]
Looks good

**Pull request merged**